assets/css/app.css

```diff
@@ -935,3 +871,73 @@ footer icons > a > svg {
         fill: #fff;
     }
 }
+
+environment {
+    display: block;
+    -webkit-touch-callout: none; /* iOS Safari */
+    -webkit-user-select: none; /* Safari */
+     -khtml-user-select: none; /* Konqueror HTML */
+       -moz-user-select: none; /* Old versions of Firefox */
+        -ms-user-select: none; /* Internet Explorer/Edge */
+            user-select: none; /* Non-prefixed version, currently
+                                  supported by Chrome, Edge, Opera and Firefox */
+}
+environment.customize menubutton  {
+    /* background: #fff; */
+}
+environment.customize menubutton span  {
+    /* color: #085078; */
+}
+environment backdrop {
+    display: none;
+    position: fixed;
+    z-index: 2;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    background: rgba(255, 255, 255, 0.7);
+}
+environment.customize backdrop {
+    display: block;
+}
+environment popup {
+    display: none;
+    position: fixed;
+    z-index: 3;
+    border: 1px solid #fff;
+    border-radius: 1vw;
+    background: linear-gradient(45deg, rgb(0, 158, 217) 0%, rgb(0, 186, 255) 50%);
+    background: linear-gradient(45deg, rgb(112, 201, 255) 10%, rgb(0, 158, 217) 80%, rgb(248, 255, 223) 100%);
+    background-attachment:fixed;
+    color: #fff;
+    top: 10vw;
+    right: 10vw;
+    bottom: 10vw;
+    left: 10vw;
+    padding: 3vw 9vw;
+}
+environment.customize popup {
+    display: block;
+}
+environment popup heading {
+    display: none;
+    color: #fff;
+    font-family: monospace;
+    font-weight: normal;
+    font-size: 2vw;
+    text-align: center;
+    margin-bottom: 2vw;
+}
+environment.customize popup heading {
```

Comment:
since the environment scene is display: none by default, this is no longer needed.